iron-lazy-page.html should import polymer.html

The following error is thrown when using new `polymer-analyzer`:
```
error:   In vendor/iron-lazy-pages/iron-lazy-page.html: [unknown-polymer-behavior] - Unable to resolve behavior `Polymer.Templatizer`. Did you import it? Is it annotated with @polymerBehavior?
```

My suggestion is that `polymer.html` should be imported explicitly by `iron-lazy-page.html`

--- a/iron-lazy-page.html
+++ b/iron-lazy-page.html
@@ -6,6 +6,8 @@ All rights reserved.
 This software may be modified and distributed under the terms
 of the BSD license.  See the LICENSE file for details.
 -->
+<link rel="import" href="../polymer/polymer.html">
+
 <!--
 Template element used in conjunction with `iron-lazy-pages`.
 

